mkdocs/docs/HPC/web_portal.md

```diff
@@ -346,3 +346,7 @@ menu item:
 
 Of course, this only affects your own web portal session (not those of
 others).
+
+## Custom apps
```

Comment:
ugent only?

Reply:
The whole web portal page is UGent-only, but we can deal with that in a separate PR if we really want to.

The custom apps are even specific to certain user groups probably, we could clearly mention that.

This PR is just to get @epauwels going with adding docs for specific custom apps